examples/next-prisma-starter/src/pages/_app.tsx

```diff
@@ -42,8 +43,9 @@ export default withTRPC(
        */
       links: [
         loggerLink({
-          enabled: ({ event }) =>
-            process.env.NODE_ENV === 'development' || event === 'error',
+          enabled: (opts) =>
+            process.env.NODE_ENV === 'development' ||
+            (opts.direction === 'down' && opts.result instanceof Error),
         }),
```

Comment:
@cyrus-za thoughts?

Reply:
I agree with switching to instanceOf Error. Not sure what the direction thing is. By down do you mean from server to cient?

Reply:
yeah exactly, you can hijack it both up and downstream